satpy/utils.py

```diff
@@ -291,6 +319,7 @@ def atmospheric_path_length_correction(data, cos_zen, limit=88., max_sza=95.):
 def get_satpos(
         data_arr: xr.DataArray,
         preference: Optional[str] = None,
+        maybe_attempt_tle: bool = False
```

Comment:
I think this parameter should be renamed to remove `maybe`, eg `use_tle`?

Reply:
Done in 2c3a5d7a.